### ISSUE TITLE
feature request : Add callback for store.shared.expires

### ISSUE THREAD

**Issue by bputt** [jxcore/jxcore#708]
When using the shared memory and expiring a key based on time, it would be extremely beneficial to allow a callback after the key expires.

For example: 

```javascript
store.shared.expires(key, timeout, function(){
 //emit event, etc...
});
```

**Comment by obastemur** [comment]
@bputt good idea. Let's see if we can make this part of the next release 0.3.1.1

**Comment by obastemur** [comment]
Unfortunately this is no go ATM. The problem is that the thread supposed to listen that callback might have unloaded already. 

One another option is to emit `keyExpired` on all the active threads.
